doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -424,50 +391,55 @@ automatically created.
          .. coqtop:: reset all
 
             Require Import List.
-            Hint Extern 5 ({?X1 = ?X2} + {?X1 <> ?X2}) => generalize  X1, X2; decide equality : eqdec.
+            Hint Extern 5 ({?X1 = ?X2} + {?X1 <> ?X2}) => generalize  X1, X2;
+              decide equality : eqdec.
```

Comment:
What about putting the newline before `generalize` so that the `ltac_expr` block is shown on a single line?